doc/integrator/backend_qgis.rst

```diff
@@ -57,45 +58,64 @@ Remove the mapserver folder:
 
    git rm mapserver
 
-Connect to Postgres from QGIS desktop
--------------------------------------
+QGIS Desktop configuration
+--------------------------
 
-This chapter is subject to change when the QGIS plugin is available.
+OWS configuration
+~~~~~~~~~~~~~~~~~
 
-Throw a tunnel
-~~~~~~~~~~~~~~
+You should setup your OWS service in the QGIS project properties in the OWS
+tab.
 
-The first solution is to create a tunnel with ssh:
+You should take care to **uncheck** the checkbox *User Layer ids as names*. If
+this checkbox is enabled you will have unreliabel layers name.
 
-.. prompt:: bash
+You should **check** the checkbox *Add geometry to feature response* in order
+to make working the WMS GetFeatureInfo correctly.
```

Comment:
which one do you prefer? The first proposition was my first attemp but Google Translate prefered mine :D